lib/request-with-oauth2.js

```diff
@@ -50,11 +50,13 @@ function get(requestGetArgs, oauthUrl, clientId, clientSecret, username, passwor
       try {
         accessToken = JSON.parse(body).access_token;
         if (_.isUndefined(accessToken)) {
-          callback(err, res, body);
+          callback(null, res, body);
         } else {
           doRequestWithAccessToken(requestGetArgs, accessToken, callback);
         }
       } catch (err2) {
+        logger.log('error', 'Problem making oauth request');
+        logger.log('error', 'Request body: ', body);
```

Comment:
These should be combined otherwise they will end up as two separate json objects in production.